aio/content/navigation.json

```
@@ -369,489 +369,489 @@
             },
             {
               "url": "guide/bootstrapping",
-              "title": "Launching Apps with a Root Module",
-              "tooltip": "Tell Angular how to construct and bootstrap the app in the root \"AppModule\"."
+              "title": "Lanzamiento de aplicaciones con un módulo raíz",
+              "tooltip": "Dígale a Angular cómo construir y arrancar la aplicación en la raíz \"AppModule\"."
             },
             {
               "url": "guide/frequent-ngmodules",
-              "title": "Frequently Used NgModules",
-              "tooltip": "Introduction to the most frequently used NgModules."
+              "title": "NgModules de uso frecuente",
+              "tooltip": "Introducción a los NgModules más utilizados."
             },
             {
               "url": "guide/module-types",
-              "title": "Types of Feature Modules",
-              "tooltip": "Description of the different types of feature modules."
+              "title": "Tipos de módulos de funciones",
+              "tooltip": "Descripción de los diferentes tipos de módulos de funciones."
             },
             {
               "url": "guide/entry-components",
-              "title": "Entry Components",
-              "tooltip": "All about entry components in Angular."
+              "title": "Componentes de entrada",
+              "tooltip": "Todo sobre componentes de entrada en Angular."
             },
             {
               "url": "guide/feature-modules",
-              "title": "Feature Modules",
-              "tooltip": "Create feature modules to organize your code."
+              "title": "Módulos de funciones",
+              "tooltip": "Cree módulos de funciones para organizar su código."
             },
             {
               "url": "guide/providers",
-              "title": "Providing Dependencies",
-              "tooltip": "Providing dependencies to NgModules."
+              "title": "Proporcionar dependencias",
+              "tooltip": "Proporcionar dependencias a NgModules."
             },
             {
               "url": "guide/singleton-services",
-              "title": "Singleton Services",
-              "tooltip": "Creating singleton services."
+              "title": "Servicios Únicos",
+              "tooltip": "Creando servicios únicos."
             },
             {
               "url": "guide/lazy-loading-ngmodules",
-              "title": "Lazy Loading Feature Modules",
-              "tooltip": "Lazy load modules to speed up your apps."
+              "title": "Módulos de funciones de carga diferida",
+              "tooltip": "Módulos de carga diferida para acelerar sus aplicaciones."
             },
             {
               "url": "guide/sharing-ngmodules",
-              "title": "Sharing NgModules",
+              "title": "Compartiendo NgModules",
               "tooltip": "Share NgModules to streamline your apps."
```

Me parece que faltó traducir esta línea
```suggestion
              "tooltip": "Comparta NgModules para optimizar sus aplicaciones."
```